app/javascript/mastodon/components/status_action_bar.js

```diff
@@ -280,7 +280,7 @@ class StatusActionBar extends ImmutablePureComponent {
     }
 
     if (writtenByMe) {
-      // menu.push({ text: intl.formatMessage(messages.edit), action: this.handleEditClick });
+      menu.push({ text: intl.formatMessage(messages.edit), action: this.handleEditClick });
```

Comment:
This should be in another PR. Preferably with support for editing media attachments.